modules/ROOT/pages/_partials/mmp-deploy-to-rtf.adoc

```diff
@@ -263,6 +263,9 @@ Configuration example:
   </http>
 </deploymentSettings>
 ----
+
+| `disableAmLogForwarding` | Disables application logs forwarding to Anypoint Monitoring. By default, it is set to `false`.
```

Comment:
What's the meaning?

- `disableAmLogForwarding` disables application-log forwarding to Anypoint Monitoring? or,
- `disableAmLogForwarding` disables application logs by forwarding them to Anypoint Monitoring?